cpp/src/grammar.hpp

```diff
@@ -33,15 +35,19 @@ class Grammar {
   // PEG parser
   peg::parser emerald_parser;
 
+public:
```

Comment:
I think this is another leftover debugging thing, it can be private actually. Removed this